maintenance/wikia/AddDiscussionsToNavigation.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+/**
+ * Maintenance script to update navigation with Discussion links.
+ * @author Michal 'slayful' Turek
+ */
+
+ini_set('display_errors', 'stderr');
+ini_set('error_reporting', E_NOTICE);
+
+require_once(dirname(__FILE__) . '/../Maintenance.php');
+
+/**
+ * Class AddDiscussionsToNavigation
+ */
+class AddDiscussionsToNavigation extends Maintenance
+{
+
+	const VAR_NAME = 'wgOasisGlobalNavigation';
+	const NAVIGATION_ELEMENT = '**Special:DiscussionsNavigation|Discussions';
+	const FORUM_NAVIGATION_ELEMENT = '**Special:Forum|Forum';
+	const REASON = 'SOC-2816';
+	const SPLIT = '\n';
+
+	private $dryRun = false;
+	private $verbose = false;
+	private $rollback = false;
+
+	public function __construct() {
+		parent::__construct();
+		$this->mDescription = "Update " . self::VAR_NAME . " with Discussions navigation "
+			. self::NAVIGATION_ELEMENT;
+		$this->addOption( 'dry-run', 'Dry run mode', false, false, 'd' );
+		$this->addOption( 'verbose', 'Show extra debugging output', false, false, 'v' );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "verbose" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)